openapi_python_client/templates/endpoint_macros.pyi

```diff
@@ -33,7 +33,7 @@ params: Dict[str, Any] = {
 }
     {% for property in endpoint.query_parameters %}
         {% if not property.required %}
```

Comment:
I think to truly make not-required the same as nullable for query params we want to add this.
```suggestion
        {% if not property.required or property.nullable %}
```